Extension/ThirdPartyNotices.Mono.txt

```diff
@@ -0,0 +1,170 @@
+

```

Comment:
This is only needed for OSX. Please do not include this in the check-in. 

I think it would be ok to add this to the .gitignore. @pieandcakes?

Reply:
@grdowns Was this added on purpose because you have a requirement for Mono now?

Reply:
No, I'm pretty sure we don't have a requirement for Mono with this change.

Reply:
This license was auto-generated. I believe Sean is correct and it will be removed.